fetch.bs

```diff
@@ -3145,58 +3148,145 @@ response <a for=/>header</a> can be used to require checking a <a for=/>request<
 Cross-Origin-Resource-Policy     = %s"same-origin" / %s"same-site" / %s"cross-origin" ; case-sensitive
 </code></pre>
 
-<p>To perform a <dfn>cross-origin resource policy check</dfn>, given a <var>request</var> and
-<var>response</var>, run these steps:</p>
+<p>To perform a <dfn>cross-origin resource policy internal check</dfn>, given a
+<a for=/>response</a> <var>response</var>, an <a for=url>origin</a>, <var>origin</var>, an
+embedder policy value <var>embedder policy value</var> and a boolean
+<var>for navigation</var>, run these steps:
```

Comment:
Since we're checking if origin, using policy X, can read a response I'd reorder the arguments a bit.

Reply:
Done.